app/view_objects/lists_presentation.rb

```diff
@@ -5,8 +5,9 @@ def initialize lists
   end
 
   def partial
-    return 'no_lists' if lists.blank?
-    'lists'
+    return 'no_matches' if lists.blank?
+    return 'lists' unless List.count.zero?
+    'no_lists'
```

Comment:
@vlasiak Does `List.count` make a call to DB? If yes, let's correct please. View object should not hit DB. Also please check [this guide](http://guides.rubyonrails.org/v3.2.21/active_record_querying.html#existence-of-objects) to see better way to check existence.